Documentation/data-collection.md

```diff
@@ -236,6 +236,10 @@ data:
     - '{__name__="che_workspace_start_time_seconds_sum"}'
     # (codeready workspaces, @ibuziuk) The overall number of attempts for starting all the workspaces.
     - '{__name__="che_workspace_start_time_seconds_count"}'
+    # (image-registry-team) v1_image_imports_total counts the number of image stream imports made
+    # using registry protocol v1, we are moving forward with deprecating this version hence we
+    # need to know how often it is used.
+    - '{__name__="v1_image_imports_total"}'
```

Comment:
Is not the full name of the metric `apiserver_v1_image_imports_total`?

Reply:
Thanks @lilic! I wasn't sure if I should or not add the "subsystem" into it, going to fix it right now.

Reply:
Name here should be as it appears in Prometheus, as this is just a match selector for the metrics (series) themselves.